plugins/alphabet-sort-renderer/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@serlo/editor-plugin-alphabet-sort-renderer",
+  "version": "0.0.0",
+  "bugs": {
+    "url": "https://github.com/serlo/editor-plugins/issues"
+  },
+  "author": "Serlo Education e.V.",
+  "license": "Apache-2.0",
+  "main": "dist/index.js",
+  "umd:main": "dist/index.umd.js",
+  "module": "dist/index.mjs",
+  "source": "src/index.ts",
+  "types": "dist/index.d.ts",
+  "repository": "serlo/editor-plugins",
+  "scripts": {
+    "prebuild": "rimraf dist",
+    "build": "splish-microbundle",
+    "build:watch": "splish-microbundle watch",
+    "ci": "tsc --noEmit",
+    "format": "prettier --write \"{{__tests__,src}/**/*,*}.{js,jsx,ts,tsx,css,scss,sass,less,json,md,markdown,yaml,yml}\""
+  },
+  "dependencies": {},
+  "devDependencies": {
+    "@types/react": "^16.0.0"
+  },
+  "peerDependencies": {
+    "@serlo/editor-ui": "^0.4.0",
```

Comment:
I think this is one of the reasons the build fails. Should be `^0.5.0`

Reply:
And why do you need `@serlo/editor-ui` in your renderer?

Reply:
Because I use styled-components

Reply:
Ah, that makes sense